recipes-go/3mdeb-rtectrl/3mdeb-rtectrl/flash.sh

```diff
@@ -0,0 +1,347 @@
+#!/bin/sh
+
+errorExit() {
+  errorMessage="$1"
+  echo "$errorMessage"
+  exit 1
+}
+
+errorCheck() {
+  errorCode=$?
+  errorMessage="$1"
+  [ "$errorCode" -ne 0 ] && errorExit "$errorMessage : ($errorCode)"
+}
+
+usage() {
+  echo "${0} [COMMAND] [OPTIONS]"
+  echo "[COMMAND]:"
+  echo "    help - show this help"
+  echo "    probe - probe flash"
+  echo "    read [FILE] - read flash to FILE"
+  echo "    write [FILE] - write flash from FILE"
+  echo "    wp-list - check WP list"
+  echo "    wp-status - check WP status"
+  echo "    wp-enable - enable WP"
+  echo "    wp-disable - disable WP"
+  echo "    wp-high - enable hardware protection of status registers"
+  echo "    wp-low -disable hardware protection of status registers"
+  echo "[OPTIONS]:"
+  echo "    -V --voltage [VOLTAGE] - specify SPI voltage to be used: 1.8V or 3.3V"
+  echo "    --flashrom-params \"[PARAMS]\" - specify additional flashrom params surrounded"
+  echo "    with double quotes"
+  echo "    -v --verbose - verbose mode"
+  echo "    -d --dry-run - run skript without executing commands"
+  echo "Examples:"
+  echo "${0} probe"
+  echo "${0} read flash.rom"
+  echo "${0} write flash.rom"
+  exit 0
+}
+
+SPI_DEV="/dev/spidev1.0"
+SPI_SPEED="16000"
+FLASHROM_PROGRAMMER="linux_spi:dev=$SPI_DEV,spispeed=$SPI_SPEED"
```

Comment:
I am not sure, whether we want to set it statically or provide it via `--flashrom-params`.

Reply:
Programmer and speed should be set statically